source/git.js

```diff
@@ -353,7 +356,7 @@ git.commit = function(repoPath, amend, message, files) {
   if (message === undefined)
     return task.setResult({ error: 'Must specify commit message' });
 
-  if ((!(files instanceof Array) || files.length == 0) && !amend)
+  if ((!(Object.prototype.toString.call(files) === '[object Array]') || files.length == 0) && !amend)
```

Comment:
Wow. That's pretty unintuitive. Still, I think we could do better here. My suggestion would be to use [`Array.isArray()`][isArray]. (Ungit doesn't support IE <9, does it?)

[isArray]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray